src/resolvers/Mutation/updateSessionTimeout.ts

```diff
@@ -0,0 +1,83 @@
+import { User } from "../../models";
+import {
+  COMMUNITY_NOT_FOUND_ERROR,
+  INVALID_TIMEOUT_RANGE,
+  USER_NOT_FOUND_ERROR,
+  UPDATE_SESSION_TIMEOUT_ARGUMENT_MISSING_ERROR,
+} from "../../constants";
+import type { MutationResolvers } from "../../types/generatedGraphQLTypes";
+import { errors, requestContext } from "../../libraries";
+import { superAdminCheck } from "../../utilities";
+import { Community } from "../../models/Community";
+
+/**
+ * Payload provided with the request for updating organization timeout.
+ */
+interface InterfaceUpdateTimeoutArgs {
```

Comment:
No need to make these types. These would be generated by CodeGen for us.

Reply:
![image](https://github.com/PalisadoesFoundation/talawa-api/assets/97614113/1efd03fb-91f3-4ecf-87cb-b84f759e3abb)

Removed the types